test/integration/agent/npl_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build linux
+// +build linux
+
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package agent
+
+import (
+	"testing"
+	"time"
+
+	nplcontroller "antrea.io/antrea/pkg/agent/nodeportlocal/k8s"
+	
+)
+
+func IntTest(t *testing.T) {
+	nplCltr, err := nplcontroller.NewTestingNPLController(61000, 62000)
+	if err != nil {
+		t.Fatalf("Failed to create testing NPLController: %v", err)
+	}
+stopCh := make(chan struct{})
+	go nplCltr.SyncRules(stopCh)
+	time.Sleep(30 * time.Second)
+	close(stopCh)
+}
```

Comment:
what is this testing?

An integration test should not take more than 1 or 2 seconds to run.

Reply:
The interval has been fixed.
Still working on adding more valid test cases in the Integration test.